core/src/main/java/org/apache/iceberg/TableMetadataParser.java

```diff
@@ -225,6 +226,12 @@ public static void toJson(TableMetadata metadata, JsonGenerator generator) throw
     }
     generator.writeEndArray();
 
+    generator.writeArrayFieldStart(STATISTICS);
```

Comment:
Can you also add deserialization? Generally when we update a parser we want to update both.

Reply:
Deserialization already exists.
https://github.com/apache/iceberg/blob/fca6101f5847ee27feaa5aeb78fe25110e1af6a9/core/src/main/java/org/apache/iceberg/TableMetadataParser.java#L466
Was added in https://github.com/apache/iceberg/pull/5450

The fact serialization was not covered was an omission.